code/game/objects/items/weapons/dice.dm

```diff
@@ -234,18 +234,38 @@
 			T.visible_message("<span class='userdanger'>Mad dosh shoots out of [src]!</span>")
 			var/turf/Start = get_turf(src)
 			for(var/direction in GLOB.alldirs)
-				var/turf/dirturf = get_step(Start,direction)
+				var/turf/dirturf = get_step(Start, direction)
 				if(prob(50))
 					new /obj/item/stack/spacecash/c1000(dirturf)
 				else
 					var/obj/item/storage/bag/money/M = new(dirturf)
-					for(var/i in 1 to rand(5,50))
+					for(var/i in 1 to rand(5, 50))
 						new /obj/item/coin/gold(M)
 		if(14)
-			//Free Gun
-			T.visible_message("<span class='userdanger'>An impressive gun appears!</span>")
+			//Tator Item
+			var/traitor_item = pick(/obj/item/chameleon,
+			/obj/item/chameleon_counterfeiter,
+			/obj/item/clothing/shoes/chameleon/noslip,
+			/obj/item/pinpointer/advpinpointer,
+			/obj/item/storage/box/syndie_kit/bonerepair,
+			/obj/item/storage/backpack/duffel/syndie/med/surgery,
+ 			/obj/item/storage/toolbox/syndicate,
+ 			/obj/item/storage/backpack/clown/syndie,
+ 			/obj/item/storage/backpack/satchel_flat,
+ 			/obj/item/camera_bug,
+			/obj/item/storage/belt/military/traitor,
+			/obj/item/clothing/glasses/chameleon/thermal,
+			/obj/item/pen/edagger,
+			/obj/item/borg/upgrade/modkit/indoors,
+			/obj/item/storage/box/syndie_kit/chameleon,
+			/obj/item/storage/box/syndie_kit/hardsuit,
+			/obj/item/implanter/storage,
+			/obj/item/clothing/gloves/fingerless/rapid,
+			/obj/item/gun/projectile/automatic/pistol,
+			/obj/item/toy/syndicateballoon)
```

Comment:
```suggestion
			var/traitor_item = pick(/obj/item/chameleon,
				/obj/item/chameleon_counterfeiter,
				/obj/item/clothing/shoes/chameleon/noslip,
				/obj/item/pinpointer/advpinpointer,
				/obj/item/storage/box/syndie_kit/bonerepair,
				/obj/item/storage/backpack/duffel/syndie/med/surgery,
 				/obj/item/storage/toolbox/syndicate,
 				/obj/item/storage/backpack/clown/syndie,
 				/obj/item/storage/backpack/satchel_flat,
 				/obj/item/camera_bug,
				/obj/item/storage/belt/military/traitor,
				/obj/item/clothing/glasses/chameleon/thermal,
				/obj/item/borg/upgrade/modkit/indoors,
				/obj/item/storage/box/syndie_kit/chameleon,
				/obj/item/storage/box/syndie_kit/hardsuit,
				/obj/item/implanter/storage,
				/obj/item/toy/syndicateballoon)
```
list needs to be indented and all high-damage weapons needs to go, miners should not be getting offensive syndie equipment.